docs/browser.md

```diff
@@ -1,25 +1,91 @@
 ---
 title: Browser Builds
-description: Guide to writing modules and optimizing browser bundles.
+description: Guide to using and writing modules and optimizing browser bundles.
 ---
 
 # Browser Builds
 
+Bitcore and most official submodules work in the browser, thanks to [browserify](http://browserify.org/) (some modules make no sense in the browser, like `bitcore-p2p`).
```

Comment:
Except bitcore-p2p can be used in a browser if it's an extension. Also this may be better as "some modules are not fully compatible with web browsers"?